docs/developer/architecture/core/uisettings-service.asciidoc

```diff
@@ -26,6 +26,16 @@ Configuration through the Advanced Settings UI is restricted to users authorised
 === Configuration with UI settings overrides
 experimental[] When a setting is configured as an override in kibana.yml, it will override any other value stored in the config saved object. If an override is misconfigured, it will fail config validation and prevent Kibana from starting up. The override applies to Kibana as a whole for all spaces and users and the option will be disabled in the Advanced Settings page. We refer to these as "global" overrides.
 
+Use the top-level `uiSettings` key for this, for example:
+
+[source,yaml]
+----
+# Less feature announcement messages
+uiSettings:
+  overrides:
+    hideAnnouncements: true
```

Comment:
This example works fine, but I wonder if it'd be better to include an example of a setting we know people commonly use overrides for? Timezone is one possibility:

```suggestion
    "dateFormat:tz": "UTC"
```

Reply:
I dig that idea. I'll update the comment to match.